### PR TITLE
Cassandra prepared statements int tests

### PR DESCRIPTION
**Pull request by conorbros**

### CODE REVIEW

**Comment by conorbros** [comment]
Seems I've found a bug in cdrs-tokio

**Comment by conorbros** [comment]
Ran `cargo update` to get the latest cdrs-tokio version.

File: shotover-proxy/tests/cassandra_int_tests/basic_driver_tests.rs [hunk 1]
```diff
@@ -983,13 +977,55 @@ APPLY BATCH;"#,
...
+    run_query(
+        session,
+        r#"BEGIN BATCH
+                INSERT INTO test_prepare_statements.table_1 (id, x, name) VALUES (1, 11, 'foo');
+                INSERT INTO test_prepare_statements.table_1 (id, x, name) VALUES (2, 12, 'bar');
+                INSERT INTO test_prepare_statements.table_1 (id, x, name) VALUES (3, 13, 'baz');
+            APPLY BATCH;"#,
+    );
+
+    let prepared = session
```
Comment (benbromhead) [inline shotover-proxy/tests/cassandra_int_tests/basic_driver_tests.rs:996]: could we do an insert prepared statement as well? Would be good to test both read and write paths on the prepared statement side of things

**Review by rukai — APPROVED** [review]
Nice!

**Review by benbromhead — APPROVED** [review]

**Pull request merged**